tools/invoke-tests

```diff
@@ -64,6 +64,9 @@ db="$build_directory/cover_db"
 
 # set Perl module include path and coverage options
 export PERL5LIB="$source_directory:$source_directory/ppmclibs/blib/lib:$source_directory/ppmclibs/blib/arch/auto/tinycv:$PERL5LIB"
+if [[ "${GITHUB_ACTIONS}" == "true" ]]; then
+    cpanm Devel::Cover
```

Comment:
I would avoid installing dependencies at this level. We should rather ensure the required package is available in the container. I'm wondering why this is suddenly required, though.

Reply:
In order to let the unhandled output be clear we need to update the Devel::Cover module and I'm aware that is not the proper way to solve the issue, but is a mitigation I've apply. As I said during the daily I'll open a new ticket to proper handle this because is not in the scope of this.

Reply:
Ok, so now we have a comment hidden in in a PR review and a ticket in future. How are we going to be reminded of this? :)
The ticket should be added in the code, and it should be at least in our Next queue.

Reply:
```suggestion
    # https://progress.opensuse.org/issues/182402
    cpanm Devel::Cover
```

Reply:
https://progress.opensuse.org/issues/182402 is now "High" and in our backlog

Reply:
Updated with the reference to the ticket